services/gemma-local/audio_processor.py

```
@@ -259,59 +267,66 @@
             # Fallback: use librosa with warnings suppressed
             with warnings.catch_warnings():
                 warnings.simplefilter("ignore")
                 audio_array, sample_rate = librosa.load(temp_file_path, sr=None, mono=True)
-            return audio_array, sample_rate
-            
+            return audio_array, int(sample_rate)
+
         except Exception as e:
             logger.error(f"Temp file loading failed: {type(e).__name__}: {e}")
-            raise ValueError(f"Unable to load audio from bytes. Temp file method failed: {e}")
+            raise ValueError(f"Unable to load audio from bytes. " f"Temp file method failed: {e}")
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)